src/tokenizer/state.js

```diff
@@ -89,8 +89,9 @@ export default class State {
   // Labels in scope.
   labels: Array<{ kind: ?("loop" | "switch"), statementStart?: number }>;
 
-  // Leading decorators.
-  decorators: Array<N.Decorator>;
+  // Leading decorators. Last element of the stack represents the decorators in current context.
+  // Supports nesting of decorators, e.g. @foo(@bar class {}) class {}
```

Comment:
I'd clarify here that `foo` applies only to the latter class and `bar` only to the inner.